app/experimenter/experiments/tests/test_admin/test_admin_nimbus.py

```diff
@@ -145,3 +159,98 @@ def test_admin_force_jetstream_fetch(self, mock_fetch_experiment_data):
         )
         self.assertEqual(response.status_code, 200)
         mock_fetch_experiment_data.delay.assert_called_with(experiment.id)
+
+
+class TestNimbusExperimentExport(TestCase):
+    def test_decimal_render(self):
+        dec_value = DecimalWidget.render(None, Decimal("90.00"))
+        self.assertEqual(dec_value, "90.00")
+        self.assertNotEqual(dec_value, 90)
+        self.assertNotEqual(dec_value, "90.0")
+
+    def test_nimbus_branch_get_queryset(self):
+        reference_branch_slug = fields.Field(
+            column_name="reference_branch_slug",
+            widget=NimbusBranchForeignKeyWidget(NimbusBranch, "slug"),
+        )
+        experiment = NimbusExperimentFactory.create()
+        experiment.id = None
+        experiment.reference_branch = None
+        test_row = {"slug": experiment.slug, "reference_branch_slug": "control"}
+        try:
+            reference_branch_slug.widget.get_queryset(value=None, row=test_row)
+        except ValueError:
+            self.fail("get_queryset raised an Exception!")
```

Comment:
Oh that's a fun little pattern, I would've just made the call and let the exception be the failure case, but this way you get to add a nice little message.  Interesting.

Reply:
Well I guess the flip side is, the exception might contain some useful information that gets elided here, maybe rather than wrapping it just let the exception fail the test itself.

Reply:
Yea I agree, especially in this case where the custom message is not really adding any value.